docs/pt/docs/tutorial/testing.md

```diff
@@ -0,0 +1,249 @@
+# Testando
+
+Graças ao <a href="https://www.starlette.io/testclient/" class="external-link" target="_blank">Starlette</a>, testar aplicativos **FastAPI** é fácil e agradável.
+
+Ele é baseado em <a href="https://www.python-httpx.org" class="external-link" target="_blank">HTTPX</a>, que por sua vez é projetado com base em Requests, por isso é muito familiar e intuitivo.
```

Comment:
I would change *"baseado em HTTPX"* to *"baseado **no** HTTPX"*

The same for requests: *"com base **no** Requests"*

It's just a suggestion ;)